core/templates/dev/head/services/StateTopAnswersStatsService.js

```diff
@@ -42,32 +54,102 @@ oppia.factory('StateTopAnswersStatsService', [
       var interactionRulesService = $injector.get(
         AngularNameService.getNameOfInteractionRulesService(
           state.interaction.id));
-      stateTopAnswerStatsCache[stateName].forEach(function(answerStats) {
+      var allAnswersCacheEntry =
+        stateTopAnswersStatsCache[stateName].allAnswers;
+      var unresolvedAnswersCacheEntry =
+        stateTopAnswersStatsCache[stateName].unresolvedAnswers;
+
+      // Clear the unresolved answers array since many answers may now have
+      // different values.
```

Comment:
different "addressed" values

or

different "addressed" statuses

(otherwise, sounds like the answer itself has changed)